### ISSUE TITLE
Add support for the maplibre-gl-compare plugin

### ISSUE THREAD

**Issue by giswqs** [eodaGmbH/py-maplibregl#66]
This is a very useful plugin. It would be great to support it.
https://github.com/maplibre/maplibre-gl-compare

**Comment by crazycapivara** [comment]
Yes, I also like, but it will not be that simple to include than the other plugins because we need a second map as overlay.
But I will check for an elegant way integrating it :-)
